node/src/bin/manager.rs

```diff
@@ -237,6 +237,29 @@ pub enum Command {
     /// General database management
     #[clap(subcommand)]
     Database(DatabaseCommand),
+
+    /// Delete a deployment and all it's indexed data
+    ///
+    /// The deployment can be specified as either a subgraph name, an IPFS
+    /// hash `Qm..`, or the database namespace `sgdNNN`. Since the same IPFS
+    /// hash can be deployed in multiple shards, it is possible to specify
+    /// the shard by adding `:shard` to the IPFS hash.
+    Drop {
+        /// The deployment identifier
+        deployment: DeploymentSearch,
+        /// List only current version
+        #[clap(long, short)]
+        current: bool,
+        /// List only pending versions
+        #[clap(long, short)]
+        pending: bool,
+        /// List only used (current and pending) versions
+        #[clap(long, short)]
+        used: bool,
+        /// Skips confirmation prompt
```

Comment:
Should be `Skip` to stay in the imperative style of the rest of the help text.

Reply:
Fixed in 7ff6aa8ba